lib/schemas/payment.js

```diff
@@ -0,0 +1,108 @@
+/**!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+const {PaymentStatus} = require('bedrock-payment');
+const {schemas} = require('bedrock-validation');
+const currencyEnum = require('./currencies.json');
+
+const numericFloat = '^[0-9]+(\\.[0-9]{0,2})?$';
```

Comment:
Why are we doing this?

Reply:
currency is a string so we need to pass it in as a string and then use this regex (it comes from another project) to validate it.